examples/XX-anomaly-detection.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.    series_taxi.plot(label="number of taxi passengers", linewidth=0.5, color="#6464ff")
</pre><p>Since there are many time steps, you could maybe make the figure wider here so it's a bit more visible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1518/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>